examples/infomax/infomax_matlab_comparison.cc

```diff
@@ -39,14 +39,14 @@ runTest(const filesystem::path &dataPath, int num)
 
     // Load matrices of weights
     const auto pref = "test"s + snum + "_"s;
-    const auto initWeights = readData<>(dataPath / (pref + "weights_init.bin"s));
-    const auto matlabOutputWeights = readData<>(dataPath / (pref + "weights_out.bin"s));
-    const auto matlabOutputWeightsMany = readData<>(dataPath / (pref + "weights_out_many.bin"));
-    const auto matlabU = readData<>(dataPath / (pref + "u.bin"s));
-    const auto matlabY = readData<>(dataPath / (pref + "y.bin"s));
+    const auto initWeights = readMatrix<double>(dataPath / (pref + "weights_init.bin"s));
+    const auto matlabOutputWeights = readMatrix<double>(dataPath / (pref + "weights_out.bin"s));
```

Comment:
I _seem_ to remember matlab binary files are column-major  - is that also the default for Eigen?

Reply:
Yeah, although I wasn't trying to make them compatible with matlab here. We're also writing the matrix size into the file first so I think that'll mess matlab up.